roles/ceph-common/defaults/main.yml

```diff
@@ -0,0 +1,155 @@
+rbd_concurrent_management_ops: 20
+rbd_client_directories: false # this will create rbd_client_log_path and rbd_client_admin_socket_path directories with proper permissions, this WON'T work if libvirt and kvm are installed
+rbd_client_log_file: /var/log/rbd-clients/qemu-guest-$pid.log # must be writable by QEMU and allowed by SELinux or AppArmor
+rbd_client_log_path: /var/log/rbd-clients/
+rbd_client_admin_socket_path: /var/run/ceph/rbd-clients # must be writable by QEMU and allowed by SELinux or AppArmor
+rbd_default_features: 3
+rbd_default_map_options: rw
+rbd_default_format: 2
+
+## Monitor options
+#
+monitor_interface: interface
+mon_osd_down_out_interval: 600
+mon_osd_min_down_reporters: 7 # number of OSDs per host + 1
+mon_clock_drift_allowed: .15
+mon_clock_drift_warn_backoff: 30
+mon_osd_full_ratio: .95
+mon_osd_nearfull_ratio: .85
+mon_osd_report_timeout: 300
+mon_pg_warn_max_per_osd: 0 # disable complains about low pgs numbers per osd
+mon_osd_allow_primary_affinity: "true"
+
+## OSD options
+#
+journal_size: 0
+pool_default_pg_num: 128
+pool_default_pgp_num: 128
+pool_default_size: 2
+pool_default_min_size: 1
+public_network: 0.0.0.0/0
+cluster_network: "{{ public_network }}"
+osd_mkfs_type: xfs
+osd_mkfs_options_xfs: -f -i size=2048
+osd_mount_options_xfs: noatime,largeio,inode64,swalloc
+osd_mon_heartbeat_interval: 30
+
+# CRUSH
+pool_default_crush_rule: 0
+osd_crush_update_on_start: "true"
+
+# Object backend
+osd_objectstore: filestore
+
+# Performance tuning
+filestore_merge_threshold: 40
+filestore_split_multiple: 8
+osd_op_threads: 8
+filestore_op_threads: 8
+filestore_max_sync_interval: 5
+osd_max_scrubs: 1
+
+# Recovery tuning
+osd_recovery_max_active: 5
+osd_max_backfills: 2
+osd_recovery_op_priority: 2
+osd_recovery_max_chunk: 1048576
+osd_recovery_threads: 1
+
+# Deep scrub
+osd_scrub_sleep: .1
+osd_disk_thread_ioprio_class: idle
+osd_disk_thread_ioprio_priority: 0
+osd_scrub_chunk_max: 5
+osd_deep_scrub_stride: 1048576
+
+## Rados Gateway options
+#
+#radosgw_dns_name: your.subdomain.tld # subdomains used by radosgw. See http://ceph.com/docs/master/radosgw/config/#enabling-subdomain-s3-calls
+radosgw_frontend: civetweb # supported options are 'apache' or 'civetweb', also edit roles/ceph-rgw/defaults/main.yml
+radosgw_civetweb_port: 80
+radosgw_keystone: false # activate OpenStack Keystone options full detail here: http://ceph.com/docs/master/radosgw/keystone/
+#radosgw_keystone_url: # url:admin_port ie: http://192.168.0.1:35357
+radosgw_keystone_admin_token: password
+radosgw_keystone_accepted_roles: Member, _member_, admin
+radosgw_keystone_token_cache_size: 10000
+radosgw_keystone_revocation_internal: 900
+radosgw_s3_auth_use_keystone: "true"
+radosgw_nss_db_path: /var/lib/ceph/radosgw/ceph-radosgw.{{ ansible_hostname }}/nss
+
+## Testing mode
+# enable this mode _only_ when you have a single node
+# if you don't want it keep the option commented
+#common_single_host_mode: true

```

Comment:
I object to the Testing mode here, we'll have valid scenarios in which we implement a single-host ceph setup.